### ISSUE TITLE
Add component owners to js-contrib-triagers role

### ISSUE THREAD

**Issue by JamieDanielson** [open-telemetry/opentelemetry-js-contrib#2600]
As a follow-up to #2548 we want to be consistent with the expectations and automations for component owners. 

This issue is for:

- [ ] Add expectations for js-contrib-triager role
- [ ] Confirm current [codeowners](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/main/.github/component_owners.yml) are members of OTel org
- [ ] Add codeowners to otel-js-contrib

**Comment by pichlermarc** [comment]
Opened #2641 to address the first point ("Add expectations for js-contrib-triager role")

**Comment by pichlermarc** [comment]
Alright, this is step 2 "Confirm current [codeowners](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/main/.github/component_owners.yml) are members of OTel org"

I took a look at the current component-owners, and it looks like the following users are not yet part of the organization:
- [x] @Ugzuzg (instrumentation-cucumber)
- [x] @abhee11 (resource-detector-container)
- [x] @d4nyll (instrumentation-runtime-node)
- [x] @henrinormak (instrumentation-dataloader)
- [x] @jacksonweber (resource-detector-azure)
- [x] @mhennoch (instrumentation-long-task)
- [x] @t2t2 (instrumentation-long-task)

Hi all, some context for the ping: we're currently adding assigning members that are [component owners](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/main/.github/component_owners.yml) in this repository to the @open-telemetry/javascript-contrib-triagers role, but to do so, users need to be members of the organization.

Adding to everyone to `CODEOWNERS` would be the ideal solution to give you all green check-marks for your components, but GitHub does not support that without also giving write permissions to the repository. Therefore, we're adding component owners to @open-telemetry/javascript-contrib-triagers, and ask owners to apply the `has:owner-approval` label when you approve a PR for your component. The goal of that is to de-facto lift component owner reviews to the same level as approver reviews (scoped to the owner's component).

The role also grants triage permissions to issues, so it should become easier for all of you to triage issues (bugs, feature-requests, ...) that are related to your component. This will also give component owners more autonomy in deciding how to handle certain requests.

[Please check here](https://github.com/open-telemetry/community/blob/main/guides/contributor/membership.md#member) for requirements and steps to become an member of the organization. I think most of you already meet the criteria there. :slightly_smiling_face:

The checklist also mentions that you need two sponsors - I've reached out to some other folks based on interactions I remember and have come up with the following list of sponsors (I already de-duped the list when there were >1 sponsors per company):

| Component Owner | Sponsors |
|-|-|
| @Ugzuzg | @blumamir, @pichlermarc |
| @abhee11 | @trentm, @pichlermarc |
| @d4nyll | @trentm, @pichlermarc |
| @henrinormak | @trentm, @pichlermarc |
| @JacksonWeber | @JamieDanielson, @pichlermarc |
| @mhennoch  | @trentm, @pichlermarc |
| @t2t2 | @JamieDanielson, @legendecas, @pichlermarc |

Once you have been added to the organization we can assign you to the role. Thanks! 🙂

**Comment by t2t2** [comment]
✅ Joined the org

**Comment by pichlermarc** [comment]
Alright I sent out the invites to everyone who's currently on the owners list - I left out current maintainers as they are already assigned to @open-telemetry/javascript-contrib-triagers due to being part of the @open-telemetry/javascript-maintainers group.

I will try to contact the ones that are still pending via other means.

For context, please see https://github.com/open-telemetry/opentelemetry-js-contrib/issues/2600#issuecomment-2595223011

| Owner | Status | Components | 
|-|-|-|
| @JacksonWeber | ✅ invite sent | resource-detector-azure |
| @hectorhdzg  | ✅ invite sent | resource-detector-azure, winston-transport |
| @jj22ee  | ✅ invite sent | resource-detector-aws, sampler-aws-xray, id-generator-aws-xray, instrumentation-aws-lambda,  instrumentation-aws-sdk, propagator-aws-xray |
| @abhee11   | ✅ invite sent | resource-detector-container | 
| @aabmass  | ✅  invite sent | resource-detector-gcp |
| @punya | ✅ invite sent | resource-detector-gcp |
| @kirrg001 | ✅ invite sent | resource-detector-instana, propagator-instana |
| @MikeGoldsmith  | ✅ invite sent | baggage-span-processor |
| @Ugzuzg  | ✅ invite sent | instrumentation-cucumber |
| @henrinormak   | ✅ invite sent | instrumentation-dataloader |
| @seemk | ✅ invite sent | instrumentation-kafkajs, instrumentation-bunyan, instrumentation-cassandra, instrumentation-net, instrumentation-pino, instrumentation-winston |
| @mottibec | ✅ invite sent | instrumentation-socketio |
| @trivikr  | ✅ invite sent | instrumentation-aws-sdk |
| @pkanal | ✅ invite sent | instrumentation-express, instrumentation-document-load | 
| @obecny | ✅ invite sent | instrumentation-graphql, instrumentation-user-interaction |
| @naseemkullah | ✅ invite sent | instrumentation-ioredis |
| @d4nyll  | ✅ invite sent | instrumentation-runtime-node |
| @maryliag | ✅ invite sent | instrumentation-pg |
| @david-luna | ✅ invite sent | instrumentation-undici |
| @martinkuba | ✅ invite sent | instrumentation-document-load, plugin-react-load |
| @t2t2  | ✅ invite sent | instrumentation-long-task |
| @mhennoch   | ✅ invite sent | instrumentation-long-task |

**Comment by JacksonWeber** [comment]
Requested membership: https://github.com/open-telemetry/community/issues/2527

**Comment by d4nyll** [comment]
Requested membership: https://github.com/open-telemetry/community/issues/2537

**Comment by trentm** [comment]
By a manual inspection, every GH user in the table above is now on the https://github.com/orgs/open-telemetry/teams/javascript-contrib-triagers/members team, with the exception of @henrinormak

**Comment by pichlermarc** [comment]
Looks like a few more folks were missing, I sent the remaining invites @open-telemetry/javascript-contrib-triagers 

| Owner | Status | Components | 
|-|-|-|
| @psx95 | ✅ invite sent | resource-detector-gcp |
| @DylanRussell | ✅ invite sent | resource-detector-gcp |
| @dashpole | ✅ invite sent | resource-detector-gcp |
| @weyert |  ✅ invite sent | instrumentation-typeorm |
| @sudarshan12s | ✅ invite sent | instrumentation-oracledb | 
| @sharadraju | ✅ invite sent | instrumentation-oracledb |

**Comment by henrinormak** [comment]
Hi, so sorry I have somehow completely missed this issue for such a long time. I see my name is mentioned multiple times, but I'm not 100% sure what am I supposed to do? I may have inadvertently missed an invite or something?

**Comment by sharadraju** [comment]
Membership request completed: https://github.com/open-telemetry/community/issues/3088

**Comment by weyert** [comment]
I think I am already part of this organisation? Happy to accept my duties for the typeorm repo

**Comment by pichlermarc** [comment]
> Hi, so sorry I have somehow completely missed this issue for such a long time. I see my name is mentioned multiple times, but I'm not 100% sure what am I supposed to do? I may have inadvertently missed an invite or something?

Hi @henrinormak, no worries - I think you're listed as a component-owner for `@opentelemetry/instrumentation-dataloader`, since you were the person that initially added it (#1171). However, we now require each component owner to be part of the OTel Organization, since component-owners are assigned triage permissions on the repository.

If you'd like to stay a component owner please [follow the steps outlined here](https://github.com/open-telemetry/community/blob/main/guides/contributor/membership.md#member) to become a member of the Organization (@trentm and myself have agreed to sponsor), then we'll then add you to the Triagers group. 🙂 

I you don't want to be a component owner, please open a PR to remove your name [here](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/5b398d7aaa8052bab90ba7138c48bf4a273f4c35/.github/component_owners.yml#L64).

**Comment by pichlermarc** [comment]
> Membership request completed: [open-telemetry/community#3088](https://github.com/open-telemetry/community/issues/3088)

Thanks, invite sent!

**Comment by pichlermarc** [comment]
> I think I am already part of this organisation? Happy to accept my duties for the typeorm repo

Yes 🙂 Now you're also member of `@open-telemetry/javascript-contrib-triagers`. 
All sorted, thanks! ✅

**Comment by henrinormak** [comment]
Opened an issue for membership - https://github.com/open-telemetry/community/issues/3159

**Comment by pichlermarc** [comment]
Thanks @henrinormak - invite sent :) 
Looks like this issue is done. Thanks everyone 🙂
